test/packages-src/bonjour/bonjour.bat

```diff
@@ -1 +1,2 @@
 ECHO bonjour!
+ECHO $CL_LOG_LEVEL
```

Comment:
I don't think this is the way to display a variable content in in windows batch scripts